StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/ReturnedFromAuthenticatedWebViewSender.swift

```diff
@@ -10,8 +10,10 @@ import Foundation
 /// Notifies that the user finished the flow within the `ASWebAuthenticationSession`
 struct ReturnedFromAuthenticatedWebViewSender: MessageSender {
     struct Payload: Codable, Equatable {
-        /// The return URL from the `ASWebAuthenticationSession` redirect. This value will be nil if the user canceled out fo the view
-        let url: String?
+        /// The return URL from the `ASWebAuthenticationSession` redirect. This value will be nil if the user canceled out of the view
+        let url: URL?
+        /// The unique identifier sent from the web view in `openAuthenticatedWebView`
+        let id: String
```

Comment:
👀 @mxl-stripe 

Added `id` so we could round-trip track the original open request